### ISSUE TITLE
Since symfony/http-foundation 6.2: The "Symfony\Component\HttpFoundation\Request::getContentType()" method is deprecated, use "getContentTypeFormat()" instead.

### ISSUE THREAD

**Issue by jeremywaguet** [scheb/2fa#169]
**Bundle version**: 6.4.0
**Symfony version**: 6.2.1
**PHP version**: 8.1.5
**Using authenticators** (`enable_authenticator_manager: true`): 
```yaml
scheb_two_factor:
    security_tokens:
        - Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken
        - Symfony\Component\Security\Http\Authenticator\Token\PostAuthenticationToken
    google:
        enabled: true
...
```

**Description**

Since symfony/http-foundation 6.2: The "Symfony\Component\HttpFoundation\Request::getContentType()" method is deprecated, use "getContentTypeFormat()" instead.

**To Reproduce**

1. logging using 2FA

**Additional Context**

Deprecation callstack:
```
{
    "class": "ErrorException",
    "message": "User Deprecated: Since symfony/http-foundation 6.2: The \"Symfony\\Component\\HttpFoundation\\Request::getContentType()\" method is deprecated, use \"getContentTypeFormat()\" instead.",
    "code": 0,
    "file": "/var/www/html/vendor/symfony/http-foundation/Request.php:1322",
    "trace": [
        "/var/www/html/vendor/scheb/2fa-bundle/Security/Http/Utils/JsonRequestUtils.php:28",
        "/var/www/html/vendor/scheb/2fa-bundle/Security/Http/Utils/RequestDataReader.php:16",
        "/var/www/html/vendor/scheb/2fa-bundle/Security/TwoFactor/TwoFactorFirewallConfig.php:106",
        "/var/www/html/vendor/scheb/2fa-bundle/Security/Http/Authenticator/TwoFactorAuthenticator.php:76",
        "/var/www/html/vendor/symfony/security-http/Authentication/AuthenticatorManager.php:171",
        "/var/www/html/vendor/symfony/security-http/Authentication/AuthenticatorManager.php:154",
        "/var/www/html/vendor/symfony/security-http/Authentication/AuthenticatorManager.php:136",
        "/var/www/html/vendor/symfony/security-http/Firewall/AuthenticatorManagerListener.php:40",
        "/var/www/html/vendor/symfony/security-http/Firewall/AbstractListener.php:26",
        "/var/www/html/vendor/symfony/security-bundle/Security/LazyFirewallContext.php:60",
        "/var/www/html/vendor/symfony/security-http/Firewall.php:116",
        "/var/www/html/vendor/symfony/security-http/Firewall.php:92",
        "/var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php:246",
        "/var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php:206",
        "/var/www/html/vendor/symfony/event-dispatcher/EventDispatcher.php:56",
        "/var/www/html/vendor/symfony/http-kernel/HttpKernel.php:139",
        "/var/www/html/vendor/symfony/http-kernel/HttpKernel.php:74",
        "/var/www/html/vendor/symfony/http-kernel/Kernel.php:184",
        "/var/www/html/vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php:35",
        "/var/www/html/vendor/autoload_runtime.php:29",
        "/var/www/html/public/index.php:5"
    ]
}
````

**Comment by scheb** [comment]
A deprecation is not a bug. It's valid use of the software. If this is bothering you, feel free to contribute a PR that makes the deprecation notice go away and ensures backwards compatibility with older Symfony versions.

**Comment by jeremywaguet** [comment]
merged with #172
